redis-test/test_strings.py

```diff
@@ -89,8 +90,12 @@ def test_select():
     r.set(key, val)
     assert r.get(key) == val
 
-    logger.debug('Switch to database 1')
-    assert r.execute_command('SELECT 1') == 'OK'
+    response = requests.get('http://127.0.0.1:10000/v2/config/redis_default_database_count')
```

Comment:
Tests are supposed to work correctly against both our code and redis (I hope that you and @fastio are verifying this, because I am not...). But this test will not work against Redis, because it doesn't have Scylla's REST API port...
In general, these Python functional tests aren't a good way to check configuration options, because they assume Redis is already running - and don't run it specially with different configuration options. So I think we can't test it like this.

Reply:
I always test with Redis manually, and the test code works with Redis server. So I can't break this by using Scylla REST API.

It's a big change to prepare the scylla-redis or real redis server by test code. We need additional PR to change this.

For testing this bug, we need to set redis_default_database_count to be larger than default 16. I will verify the count by `select 16` command. If it succeeded, it means the configuration is correct, and we can try to write data to trigger the bug, or verify the bug is fixed.

Reply:
Hi @nyh, just updated the tests, removed the Scylla API usage, and keep all tests work with native Redis server.